releng/com.zeligsoft.dds4ccm.update.axcioma/rpm_build/zeligsoftCX_axcioma.spec

```diff
@@ -95,6 +95,7 @@ elif [[ ${atcdInstalledFeature} != "" ]]; then 		# atcd is already installed
    -uninstallIU org.eclipse.cdt.native.feature.group \
    -uninstallIU org.eclipse.cdt.platform.feature.group \
    -uninstallIU org.eclipse.cdt.sdk.feature.group \
+   -uninstallIU org.eclipse.papyrus.compare.feature.feature.group \
```

Comment:
This should be up in the axcioma portion of the if/else block. Not sure we really need an if/else here at all if the code is the same between the two...that's a question for @elder4p